lambda_resources.tf

```
@@ -0,0 +1,31 @@
+locals {
+  self_signed_certificate_lambda_code_version = "v0.1.0"
+  self_signed_certificate_sidecar_hosts = "${var.sidecar_dns_name}" != "" ? (
+    "${var.sidecar_dns_name}"
+    ) : (
+    "poc.cyral.com"
+  )
+}
+
+resource "aws_lambda_function" "self_signed_certificate_generator" {
+  function_name = "${var.name_prefix}-self_signed_certificate_generator"
+  role          = aws_iam_role.self_signed_certificate_lambda_execution.arn
+  runtime       = "go1.x"
+  handler       = "selfsigned-lambda"
+  timeout       = 120
+  s3_bucket     = "cyral-public-assets-${data.aws_arn.cw_lg.region}"
+  s3_key        = "sidecar-certificate-selfsigned/${local.self_signed_certificate_lambda_code_version}/sidecar-certificate-selfsigned-lambda-${local.self_signed_certificate_lambda_code_version}.zip"
+
+  environment {
+    variables = {
+      SIDECAR_SELFSIGNED_CERTIFICATE_AWS_REGION    = data.aws_arn.cw_lg.region
+      SIDECAR_SELFSIGNED_CERTIFICATE_SIDECAR_ID    = var.sidecar_id
+      SIDECAR_SELFSIGNED_CERTIFICATE_SIDECAR_HOSTS = local.self_signed_certificate_sidecar_hosts
```

Since the service name is [sidecar-certificate-selfsigned](https://github.com/cyralinc/sidecar-certificate-selfsigned/blob/main/internal/config/config.go#L21), shouldn't these env vars have the following prefix instead?
```suggestion
      SIDECAR_CERTIFICATE_SELFSIGNED_AWS_REGION    = data.aws_arn.cw_lg.region
      SIDECAR_CERTIFICATE_SELFSIGNED_SIDECAR_ID    = var.sidecar_id
      SIDECAR_CERTIFICATE_SELFSIGNED_SIDECAR_HOSTS = local.self_signed_certificate_sidecar_hosts
```

---

Thanks Victor!